app/services/ems_dashboard_service.rb

```diff
@@ -33,15 +33,15 @@ def recent_resources(model, title, label)
     }.compact
   end
 
+  CREATED_ON_SELECT = %{to_char("%<table>s"."created_on", 'YYYY-MM-DD') as created_on_fmt}.freeze
```

Comment:
I don't really understand the embedding of the table with the trailing s, when it already has the trailing s in it.  Perhaps just a typo?

Even, so, I *think* @djberg96 's adjustment avoids having to get into the SQL specific fragments at all and the need for connection.select_all, which is preferable if it works.

Reply:
Ohhhh nvm, this is an extension to `%s` for printf... wow that's confusing 😆 ... please use the `%{table}` interpolation format instead  (I think rubocop flagged on this too)

Reply:
Don't get me started on this... I rebased this PR 3-4 times on this part alone already...

It was late last night, so I said "screw it" and was going to tackle it in the morning...

Reply:
Looking back on this, I should have used these methods:

https://api.rubyonrails.org/classes/ActiveRecord/Sanitization/ClassMethods.html

So I will at least switch this to that since it is an easy change... and I should have been SQL injection minded from the get go... crap...